gpr_unit_provider.gpr

```diff
@@ -26,7 +26,7 @@ with "libadalang";
 project gpr_unit_provider is
 
    type Build_Type is ("debug", "release", "release_checks");
-   Build : Build_Type := external ("BUILD", "debug");
+   Build : Build_Type := external ("GPR_UNIT_PROVIDER_BUILD", "debug");
```

Comment:
The pattern of chained scenario variables should be used instead.

Reply:
Well, when I did "chained scenario variables" for [gpr2](https://github.com/AdaCore/gpr/commit/fb04bceed9fd2ffd26cb10639d85913591f4bad4) I was requested to remove it.